.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -22,6 +22,7 @@ labels: task
 * [ ] Build and upload conda packages on buildfarm (check "upload").
 * [ ] Build wheels (`$PYTHON_VERSIONS`) on the buildfarm.
 * [ ] Verify packages uploaded to Anaconda Cloud and move to `numba/label/main`.
+* [ ] Build sdist locally using `python setup.py sdist --user=ci --group=numba` with umask `0022`.
```

Comment:
```suggestion
* [ ] Build sdist locally using `python setup.py sdist -u=ci --group=numba` with umask `0022`.
```

Reply:
should we then also use `-g` for coherence?

Reply:
Am happy with either, though think that `-g` does make it more consistent. The most important thing is to fix the `--user` part as it seems like that doesn't exist!

Reply:
`--owner` sounds good RE obviousness!